src/core/qgsproject.h

```diff
@@ -969,7 +998,7 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
     /**
      * Emitted when a project is being read.
      */
-    void readProject( const QDomDocument & );
+    void readProject( const QDomDocument &, QgsReadWriteContext &context );
```

Comment:
I'm unsure - is this an api break? It could cause issues with existing connections